tests/metrics/test_categorization_metrics.py

```diff
@@ -0,0 +1,84 @@
+from sklearn.metrics import f1_score
+
+from nucleus import CategoryAnnotation, CategoryPrediction
+from nucleus.metrics.categorization_metrics import CategorizationF1
+from tests.helpers import TEST_CATEGORY_ANNOTATIONS
+
+
+def test_perfect_match_f1_score():
+    annotations = [
+        CategoryAnnotation.from_json(p) for p in TEST_CATEGORY_ANNOTATIONS
+    ]
+    predictions = [
+        CategoryPrediction(label=ann.label, reference_id=ann.reference_id)
+        for ann in annotations
+    ]
+    metric = CategorizationF1()
+    results = []
+    for ann, pred in zip(annotations, predictions):
+        results.append(metric.eval([ann], [pred]))
+
+    assert results
+    aggregate_result = metric.aggregate(results)
+    assert aggregate_result.value == 1
+
+
+def test_no_match_f1_score():
+    annotations = [
+        CategoryAnnotation.from_json(p) for p in TEST_CATEGORY_ANNOTATIONS
+    ]
+    predictions = [
+        CategoryPrediction(label="sth_else", reference_id=ann.reference_id)
+        for ann in annotations
+    ]
+    metric = CategorizationF1()
+    results = []
+    for ann, pred in zip(annotations, predictions):
+        results.append(metric.eval([ann], [pred]))
+
+    assert results
+    aggregate_result = metric.aggregate(results)
```

Comment:
I think it's important to be able to call the metric directly and use the `__call__` interface if possible so that the metrics all have the same interface.

Reply:
Will change!